### ISSUE TITLE
add a push notification in app

### ISSUE THREAD

**Issue by douglasffilho** [avuenja/tabnews-app#4]
As a suggestion of app improvement, what about to add an in app push notification for every new post?

**Comment by meccin** [comment]
Certainly, after we make the MVP version, the all suggestions will are analysed and added in Milestones!

Thank, by the contribute!
